compiler/rustc_parse/src/parser/attr.rs

```diff
@@ -373,7 +373,7 @@ impl<'a> Parser<'a> {
     /// MetaItem = SimplePath ( '=' UNSUFFIXED_LIT | '(' MetaSeq? ')' )? ;
     /// MetaSeq = MetaItemInner (',' MetaItemInner)* ','? ;
     /// ```
-    pub fn parse_meta_item(&mut self) -> PResult<'a, ast::MetaItem> {
+    pub fn parse_meta_item(&mut self, allow_leading_unsafe: bool) -> PResult<'a, ast::MetaItem> {
```

Comment:
Instead of a `bool`, can we make this an `enum AllowLeadingUnsafe { Yes, No }`? It makes it easier to read the *calls* in isolation without having to check the signature.